src/Propel/Generator/Model/Table.php

```diff
@@ -2074,9 +2074,9 @@ public function __clone()
         foreach ($this->columns as $oldCol) {
             $col = clone $oldCol;
             $columns[] = $col;
-            $this->columnsByName[$col->getName()] = $col;
-            $this->columnsByLowercaseName[strtolower($col->getName())] = $col;
-            $this->columnsByPhpName[$col->getPhpName()] = $col;
+            $this->columnsByName[(string)$col->getName()] = $col;
```

Comment:
Are these casts really needed or were they from WIP commits?

Reply:
As long as we dont have a get*OrFail() method ( ?string returned ).